libtenzir/builtins/contexts/bloom_filter.cpp

```diff
@@ -91,6 +91,31 @@ class bloom_filter_context final : public virtual context {
     };
   }
 
+  auto dump() -> generator<table_slice> override {
+    auto ptr = reinterpret_cast<const std::byte*>(bloom_filter_.data().data());
+    auto size = bloom_filter_.data().size();
+    auto data = std::basic_string<std::byte>{ptr, size};
+    auto entry_builder = series_builder{};
+    auto row = entry_builder.record();
+    row.field("num_elements", bloom_filter_.num_elements());
+    auto params = row.field("parameters").record();
+    if (bloom_filter_.parameters().m) {
+      params.field("m", *bloom_filter_.parameters().m);
+    }
+    if (bloom_filter_.parameters().n) {
+      params.field("n", *bloom_filter_.parameters().n);
+    }
+    if (bloom_filter_.parameters().p) {
+      params.field("p", *bloom_filter_.parameters().p);
+    }
+    if (bloom_filter_.parameters().k) {
+      params.field("k", *bloom_filter_.parameters().k);
+    }
+    row.field("data", data);
+    co_yield entry_builder.finish_assert_one_slice(dump_event_name());
+    co_return;
```

Comment:
```suggestion
```

Reply:
Removed in https://github.com/tenzir/tenzir/pull/3893/commits/99652cea4c33742e6f0b7e36b1046368493dc474